README.md

```diff
@@ -83,7 +83,7 @@ Or you can exclude your particular custom post type from the confirmation dialog
 );</pre>
 
 ### Is there any way to change the default dialogue message into something else? ###
-The message text in the publishing dialogue can be changed via PHP filter from a custom plugin or your theme’s functions.php:
+The message text in the publishing dialogue can be changed via PHP filter from a custom plugin or your theme's functions.php:
```

Comment:
Yes, a [typographic apostrophe](http://practicaltypography.com/apostrophes.html).
Never mind, it’s an old discussion. :)